schematic/store/synapse.py

```diff
@@ -484,11 +484,16 @@ def associateMetadataWithFiles(
             #  prepare metadata for Synapse storage (resolve display name into a name that Synapse annotations support (e.g no spaces)
             metadataSyn = {}
             for k, v in row.to_dict().items():
+
                 if useSchemaLabel:
                     keySyn = se.get_class_label_from_display_name(str(k))
                 else:
                     keySyn = str(k)
 
+                # Skip `Filename` and `ETag` columns when setting annotations
+                if keySyn in ["Filename", "ETag"]:
```

Comment:
Hey @BrunoGrandePhD, this works for `Filename`, but I wasn't able to get this to work in the NF app for `eTag` - but in the NF app, `useSchemaLabel = T`. I also noticed that entityId was also being set as an annotation - I missed that in the original issue. 

I tried changing L494 to this: 

```
                if keySyn in ["Filename", "ETag", "eTag", "EntityId", "entityId"]:
```
but both eTag and entityId seem to be annotated. Not sure why this approach is working for Filename, but not the other values.

Reply:
I did notice `EntityId` being set as an annotation, but that's been the case as far as I can remember. I confirmed this with files that were annotated in December before we deployed the `--use_annotations` feature. I hesitated to drop the `EntityId` annotation because I would need to confirm that it's not being used downstream (_e.g._ when updating metadata).

Reply:
I'm not sure why `eTag` is sneaking through. 

@allaway: Can you post the commands that you're running to reproduce the issue? I'll check what the values of `keySyn` end up being so I can figure out what's going on. Thanks!

Reply:
@allaway and @BrunoGrandePhD we carry entityId as annotations in the metadata manifest exactly for the reason @BrunoGrandePhD pointed:  we can update metadata and support other downstream functionality (e.g. getting s3 locations of file entities on Synapse, so that we can let other services process the files). I see that storing entityId as annotations on a file in Synapse is pretty redundant. However, before we drop it let's make sure we have all necessary logic to pull entityId's when/if we are populating csv manifests/google sheets using file annotations.